server/migrations/sql/1_create_table.sql

```diff
@@ -0,0 +1,38 @@
+-- uuid_generate_v4() extension
+-- create extension if not exists "uuid-ossp";
+CREATE TABLE users(
+    id uuid PRIMARY KEY DEFAULT
+    uuid_generate_v4(),
+    user_name VARCHAR(255) NOT NULL,
+    user_email VARCHAR(255) NOT NULL,
+    user_password VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE blogs(
+    id uuid PRIMARY KEY DEFAULT
+    uuid_generate_v4(),
+    user_id VARCHAR(255) NOT NULL,
+    user_name VARCHAR(255) NOT NULL,
+    question VARCHAR(255) NOT NULL,
+    blog_text VARCHAR(255) NOT NULL,
+    blog_date  VARCHAR(255) NOT NULL
+);
+-- reactions is table for upvotes and downvotes
+CREATE TABLE reactions(
+    id uuid PRIMARY KEY DEFAULT
+    uuid_generate_v4(),
+    blog_id VARCHAR(255),
+    comment_id VARCHAR(255),
+    user_id VARCHAR(255) NOT NULL,
+    reaction_type VARCHAR(255) NOT NULL,
```

Comment:
As we stated before if something will have certain amount of values it should use something like this: https://www.postgresql.org/docs/current/datatype-enum.html